ui/src/routes.ts

```diff
@@ -81,4 +82,11 @@ export const routes: Array<Route> = [
     enabled: true,
     component: ProfileContainer,
   },
+  {
+    key: 'songsView-route',
+    title: 'songsView',
+    path: '/song/:id',
```

Comment:
we have been doing `/songs` for all songs related routes. can we make it consistent? either all /songs or all /song